Assets/LeapMotion/Scripts/LeapServiceProvider.cs

```diff
@@ -142,7 +142,7 @@ public class LeapServiceProvider : LeapProvider {
 
         // TODO: Add baseline & offset when included in API
         // NOTE: Alternative is to use device type since all parameters are invariant
-        info.isEmbedded = devices[0].IsEmbedded;
+        info.isEmbedded = devices[0].Type == Device.DeviceType.TYPE_KEYBOARD || devices[0].Type == Device.DeviceType.TYPE_LAPTOP;
```

Comment:
That sounds good to me.  I guess there should be a revamp for device info at some point, so we can stop hacking around, but I think that all of this should be in a separate PR, since it needs wider changes like fixing Disconnection Notice